meson.build

```diff
@@ -0,0 +1,74 @@
+project ('webview', ['c', 'cpp'])
+
+compiler_c = meson.get_compiler('c')
+compiler_cpp = meson.get_compiler('cpp')
+
+# platform specific dependenies
+webview_dependencies = []
+host_os = host_machine.system().to_lower()
+if host_os.contains('windows')

```

Comment:
You don't need to_lower() or contains: https://mesonbuild.com/Reference-tables.html#operating-system-names

Reply:
I know nothing about Meson, but it should be as simple as `if host_machine.system() == 'windows'`, as @eli-schwartz notes. It also is used in the documentation here: https://mesonbuild.com/howtox.html#generate-configuration-data-from-files